superset-frontend/src/explore/components/DataTableControl/index.tsx

```diff
@@ -306,6 +311,9 @@ export const useTableColumns = (
                   if (timeFormattedColumnIndex > -1) {
                     return timeFormatter(value);
                   }
+                  if (value === null) {
```

Comment:
I think this should be before the timeFormatter, if the value is NULL it might crash it.
Can you test it out? (pass null directly or force it somehow, to see what happens there)

Reply:
@diegomedina248 
Resolved.